src/uu/mv/src/mv.rs

```diff
@@ -103,6 +104,14 @@ static OPT_VERBOSE: &str = "verbose";
 static OPT_PROGRESS: &str = "progress";
 static ARG_FILES: &str = "files";
 
+/// Returns true if the passed `path` ends with a path terminator.
+fn path_ends_with_terminator(path: &Path) -> bool {
+    path.as_os_str()
+        .as_bytes()
+        .last()
+        .map_or(false, |&byte| std::path::is_separator(byte as char))
+}
+
```

Comment:
Is something like this already implemented somewhere?

Reply:
I'm not aware of such a function and I couldn't fine one.